README.md

```diff
@@ -80,7 +80,7 @@ script, or set via docker environment variables.
 | **DEFAULT\_ACCOUNT\_ID** | String | | The default account ID to assume roles in, if IAM\_ROLE does not contain account information. If unset, metadataproxy will attempt to lookup role ARNs using iam:GetRole. |
 | **ROLE\_SESSION\_KEY** | String | | Optional key in container labels or environment variables to use for role session name. Prefix with `Labels:` or `Env:` respectively to indicate where key should be found. Useful to pass through metadata such as a CI job ID or launching user for audit purposes, as the role session name is included in the ARN that appears in access logs. |
 | DEBUG | Boolean | False | Enable debug mode. You should not do this in production as it will leak IAM credentials into your logs |
-| DOCKER\_URL | String | unix://var/run/docker.sock | Url of the docker daemon. The default is to access docker via its socket. |
+| DOCKER\_URL | String | unix://var/run/docker_sockets/docker.sock | Url of the docker daemon. The default is to access docker via its socket. |
```

Comment:
Hm. I have worries about changing this for downstream users. They'll upgrade and all of a sudden their entire infra will fail.